<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_selectedROI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>576</width>
      <height>324</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_mainObj">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>576</width>
      <height>324</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_loadImage">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Incarca imaginea</string>
    </property>
   </widget>
   <widget class="QPushButton" name="brn_chooseBackgroundColor">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Alege culoarea de fundal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_saveImage">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>140</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvare imagine</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_choosenBackgroundColor">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_thresholdValue">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>200</y>
      <width>16</width>
      <height>221</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_thresholded">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>60</y>
      <width>190</width>
      <height>280</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_dilated">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>60</y>
      <width>190</width>
      <height>280</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_eroded">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>380</y>
      <width>190</width>
      <height>280</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_newROI">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>380</y>
      <width>190</width>
      <height>280</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Thresholded">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="Dilated">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dilatata</string>
    </property>
   </widget>
   <widget class="QLabel" name="Eroded">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>350</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Dilatata -&gt;Erodata</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1213</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuT05">
    <property name="title">
     <string>T05</string>
    </property>
   </widget>
   <addaction name="menuT05"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
